packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx

```diff
@@ -33,28 +34,42 @@ const getFilteredFixtures = (fixtures, searchText) => {
   }, {});
 };
 
+const isFixtureSelected = (fixtures, component, fixture) => {
+  return (
+    component && fixture &&
+    fixtures[component] &&
+    fixtures[component].indexOf(fixture) !== -1
+  );  
+}
+
 export default class FixtureList extends Component {
   state = {
     searchText: '',
   };
 
   componentDidMount() {
     window.addEventListener('keydown', this.onWindowKey);
-  }
+    const {fixtures, urlParams, router} = this.props;
 
-  componentWillUnmount() {
-    window.removeEventListener('keydown', this.onWindowKey);
+    if (isFixtureSelected(fixtures, urlParams.component, urlParams.fixture)) {
+      findDOMNode(this.refs[`componentName-${urlParams.component}`])
+        .scrollIntoView({ behavior: 'smooth' });
+    } 
   }
 
-  onWindowKey = e => {
-    const isFocused = this.searchInput === document.activeElement;
+    componentWillUnmount() {
+      window.removeEventListener('keydown', this.onWindowKey);
+    }
+
+    onWindowKey = e => {
+      const isFocused = this.searchInput === document.activeElement;
 
-    if (e.keyCode === KEY_S && !isFocused) {
-      // Prevent entering `s` in the search field along with focusing
-      e.preventDefault();
+      if (e.keyCode === KEY_S && !isFocused) {
```

Comment:
You must've indented these lines by mistake. Please remove one level of indentation and **make sure `yarn run lint` doesn't fail** – you can see the build is failing because of this:

![image](https://user-images.githubusercontent.com/250750/28976424-1674a3ae-793e-11e7-8859-9dea54c81e5d.png)
![image](https://user-images.githubusercontent.com/250750/28976439-2f8ec9fa-793e-11e7-96f8-ef4225972d47.png)